clicore/runner.go

```diff
@@ -0,0 +1,72 @@
+package clicore
+
+import (
+	"context"
+
+	"github.com/solo-io/go-utils/contextutils"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+type CommandConfig struct {
+	Args                string
+	Command             func(context.Context, string) *cobra.Command
```

Comment:
nit: define a type for this function

Reply:
good idea, added https://github.com/solo-io/glooshot/pull/22